### PR TITLE
Add redis to SimpleBook

### LINKED ISSUE #123: Add redis to simplebook ansible
SimpleBook relies on redis (whether this makes sense in the long run is another question).

I did not include redis in the original ansibles, meaning they're incomplete right now.

### PR DESCRIPTION
**Pull request by slifty**
This installs and configures redis, which is a SimpleBook dependency.

Resolves #123

### CODE REVIEW

File: base/simplebook/ansible/roles/simplebook/tasks/main.yml [hunk 1]
```diff
@@ -54,9 +55,30 @@
...
+- name: Restart redis
+  become_user: root
+  shell:
+    systemctl restart redis
+
+- name: Prepare NodeJS
+  become_user: root
   shell: |
-    curl -sL https://deb.nodesource.com/setup_14.x | sudo -E bash - && sudo apt-get install -y nodejs
+    curl -sL https://deb.nodesource.com/setup_14.x | bash -
```
Comment (slifty) [inline base/simplebook/ansible/roles/simplebook/tasks/main.yml:73]: This should be a separate commit -- I removed use of sudo since it made ansible hang / that is not the ansible way of using sudo.

Comment (frankduncan) [reply base/simplebook/ansible/roles/simplebook/tasks/main.yml:73]: Yeah, but if it doesn't end up happening, just add an addendum to the commit message "also slipped into this change, X"

**Review by frankduncan — CHANGES_REQUESTED** [review]
I don't think we need to check in a full redis conf that has all the comments for everything.  Might be better to just do the configurations we need, and then "See this place on the internet for the expanded conf file" as a comment at the top.  Though maybe you can see us wanting to have that information at our fingertips later.

**Comment by slifty** [comment]
I actually didn't need a copy of the conf at all -- that was from an earlier installation that didn't use regex replacement, and I forgot to delete the template.
